scripts/habitat_dataset_processing/README.md

```diff
@@ -0,0 +1,190 @@
+# Habitat Dataset Processing
+
+Asset pipeline for processing Habitat datasets for external engines (e.g. Unity).
+
+The processed datasets retain the directory structure of the input (e.g. `data/dataset/asset.obj`), enabling replay rendering in other engines.
+
+- [Installation](#installation)
+  - [Magnum Installation](#magnum-installation)
+    - [MacOS](#macos)
+    - [Linux](#linux)
+- [Usage](#usage)
+  - [Example usage:](#example-usage)
+  - [Parameters](#parameters)
+  - [Script Configuration](#script-configuration)
+  - [Data Sources](#data-sources)
+    - [AssetSource](#assetsource)
+    - [HabitatDatasetSource](#habitatdatasetsource)
+  - [metadata.json](#metadatajson)
+    - [Group types](#group-types)
+
+
+## Installation
+
+1. Create a new conda environments for dataset processing.
+
+```
+conda create -n datasets python=3.10
+conda activate datasets
+```
+
+2. Install the project.
+
+```
+cd scripts/habitat_dataset_processing
```

Comment:
A bit odd to see a python package under `scripts/`. I think of scripts as standalone utilities that live outside packages.

Reply:
I agree, `scripts/` isn't the right place for this. Where would you relocate this package?

I'm a bit wary of pushing top-level directories without a track record of full adoption.